Monika After Story/game/event-handler.rpy

```diff
@@ -1489,21 +1489,22 @@ init python:
         else:
             return False
 
-
     def restartEvent():
-        #
-        # This checks if there is a persistent topic, and if there was push it
-        # back on the stack with a little comment.
-        #
-        # IN:
-        #
-        if not mas_isRstBlk(persistent.current_monikatopic):
-            #don't push greetings back on the stack
-            pushEvent(persistent.current_monikatopic)
-            pushEvent('continue_event')
-            persistent.current_monikatopic = 0
-        return
-
+            #
+            # This checks if there is a persistent topic, and if there was push it
+            # back on the stack with a little comment.
+            #
+            # IN:
+            #
+            if not mas_isRstBlk(persistent.current_monikatopic):
```

Comment:
should just add `inactivity_detection` to the restart blacklist. (`RESTART_BLKLST`), then you wouldnt have to do the starts with check.